woocommerce/woocommerce
# Remove WCPay Subscriptions offer page experiment code

### All Submissions:






### Changes proposed in this Pull Request:



Closes #34275.

This PR removes the code and functionality introduced for the WCPay Subscriptions offer page experiment that has now ended.

In https://github.com/woocommerce/woocommerce/pull/32957 we added an admin page to **WooCommerce → Subscriptions** as an experiment to promote WooCommerce Payments to eligible merchants. This experiment has ended and as such, we need to remove the experiment code from WooCommerce Core.

### How to test the changes in this Pull Request:

<details>
<summary>1. Ensure your store is eligible to view the WC → Subscriptions experiment page. (see collapsed content for instructions)</summary>


1. **Plugin eligibility**: Ensure the following plugins are not installed at all: 
    - WooCommerce Subscriptions
    - WooCommerce Payments
    - Sumo Subscriptions
    - Yith Subscriptions
    - Subscriptio
    - Subscriptions For WooCommerce
3. **Store tracking eligibility:** Your store allows tracking.
    - This checkbox should be checked: **WooCommerce → Settings → Advanced → WooCommerce.com → Enable tracking**.
3. **Store location eligibility:** For the menu item to be displayed your store needs to based in the US. 
    - Go to **WooCommerce > Settings** and make sure your store is set to a US based address. 
3. **Store age criteria:** To view the subscription menu item your store will need to have been active for more than 6 months - we're testing this experiment on established stores. If your store doesn't meet this criteria, you can use the following code snippet: 
    - `update_option( 'woocommerce_admin_install_timestamp', strtotime( '-6 month' ) );`
    - **_or_** change the `woocommerce_admin_install_timestamp` option to `1620866357` (timestamp from ~1 year ago) via the `/wp-admin/options.php` page.
5. **Paid order criteria:** Only stores with at least 1 order in the last 30 days are eligible. If your store doesn't have an order in the last 30 days. Either: 
    1. Set any existing pending or failed order to processing or completed 
    3. If you don't have any order at all, you can just manually create one in **WooCommerce > Orders > Add new** and set it to processing or completed status
    4. ⚠️ **Note:** this order sale query is cached and so you may need to delete the `'woocommerce-wcpay-subscriptions_recent_sales_eligibility'` transient if this criteria failed previously. 
</details>

2. Checkout the `trunk` branch. If you meet all the criteria above you should see a **WooCommerce > Subscriptions (new)** menu item.
3. Checkout this PR branch and run `TURBO_FORCE=true pnpm build`. The **WooCommerce > Subscriptions (new)** menu item and page will no longer be visible.

### Other information:

-   [x] Have you written new tests for your changes, as applicable? (Not applicable)



### FOR PR REVIEWER ONLY:
